crates/rpc/rpc-types/src/eth/trace/parity.rs

```diff
@@ -327,6 +327,7 @@ pub struct MemoryDelta {
 
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
+/// A diff of some storage value.
```

Comment:
stlye nit: move above derives

Reply:
I am also looking into `VMOperation` and members `op` and `idx`.

[Ethers-rs](https://github.com/gakonst/ethers-rs/blob/030bf439a100dcacd2e968e3114de0612229056b/ethers-core/src/types/trace/mod.rs#L137) has it as an enum of OpCodes while `[erigon](https://github.com/ledgerwatch/erigon/blob/9d3551f8cf3f2835ba8212473024a609a53e24f7/turbo/jsonrpc/trace_adhoc.go#L112C2-L113C40)` has both `op` and `idx`.

 I wanted to fix this in one PR, but can make another if you like.